utils/lzbench/Makefile

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2025 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
```suggestion
```

Lets drop it according to CONTRIBUTING.md file, which is in this repo.

Reply:
applied